### PR TITLE
Feature/manage league

### PR DESCRIPTION
**Pull request by oacortez**
What (if any) features are you implementing?
- Add a manage component that are able to add players
- Add api calls to GET and POST request
- Create a modal component for error handling
- Style modal and done button on manage component
- Add error handling

What (if anything) did you refactor?
- n/a

Were there any issues that arose?
- n/a

Is there anything that you need from your teammate?
- Please review my PR

Any other comments, questions, or concerns?
-n/a

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,7 +18,8 @@ const App = () => {
       par: 54,
       holes: 18,
     },
-    courses: []
+    courses: [],
+    error: null
```
Comment (philalewis) [inline src/App.js:22]: It ended up being important that we set this to null instead of an empty string.

File: src/Components/Manage.js [hunk 2]
```diff
@@ -1,8 +1,58 @@
...
+    event.preventDefault()
+    addNewPlayer(nameInput)
+    .then(data => {
+      setDiscContext({leagueMembers: [...discContext.leagueMembers, data]})
+      setNameInput('')
+    })
+    .catch(error => setDiscContext({error: error}))
+  }
+
+  useEffect(() => {
```
Comment (philalewis) [inline src/Components/Manage.js:26]: This will be invoked every time the global state changes

**Review by philalewis — APPROVED** [review]

File: src/apiCalls.js [hunk 3]
```diff
@@ -1,7 +1,35 @@
-
+const handleError = (response) => {
+  if(!response.ok) {
+    throw `${response.status} ${response.statusText}`
```
Comment (GraceGardner) [inline src/apiCalls.js:3]: Like this dynamic error handling

File: src/Styles/ErrorModal.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgba(0,0,0,0.4);
+  display: flex;
+  justify-content: space-between;
+}
+
+.modal-box {
+  display: flex;
```
Comment (GraceGardner) [inline src/Styles/ErrorModal.scss:18]: Might be able to turn some of this into mixin's

**Review by GraceGardner — APPROVED** [review]
Good start!

File: src/apiCalls.js [hunk 5]
```diff
@@ -1,7 +1,35 @@
...
+export const addNewPlayer = (name) => {
+
+  return fetch('https://discify-api.herokuapp.com/api/v1/players', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({
+      player: {
+        name: name
```
Comment (robbiejaeger) [inline src/apiCalls.js:24]: For instances like this where the property is the same and the variable holding the value, you can use object property value shorthand. `player: { name }`

**Pull request merged**